dashboards-observability/.cypress/integration/VisualizationCharts/10_scatter_chart.spec.js

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import {
+    delay,
+    TEST_QUERIES,
+    querySearch,
+    landOnEventVisualizations
+  } from '../../utils/event_constants';
+  
+  const numberOfWindow = 4;
```

Comment:
Can we move all these constants to the corresponding constant file under .cypress/utils?

Reply:
currently for slider and input field we have different default value for different chart (Issue 896), once that gets updated we can move all constants to a common file under .cypress/utils